acra-censor/handlers/handlers_util.go

```diff
@@ -58,3 +62,15 @@ func TrimStringToN(query string, n int) string {
 	}
 	return query[:n]
 }
+
+func RedactSQLQuery(sql string) (string, error) {
```

Comment:
comment missing

"// RedactSQLQuery returns a sql string with the params stripped out for display. Taken from sqlparser package"